applications/openshift/api-server/api_server_tls_security_profile_not_old/kubernetes/shared.yml

```diff
@@ -0,0 +1,10 @@
+# platform = multi_platform_ocp
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+    name: cluster
+spec:
+    tlsSecurityProfile:
+        intermediate: {}
+        type: Intermediate
```

Comment:
I think we should stick to the defaults here as well, if we provide any remediation